test/e2e/specs/all-features/record/related-table.spec.ts

```diff
@@ -544,7 +528,7 @@ test.describe('Related tables', () => {
         rowViewPaths: [
           [{ column: 'id', value: '1' }], [{ column: 'id', value: '3' }]
         ],
-        count: 2, // by load time it's one but when we add another related for the other table this should be updated too.
+        count: 1, // by load time it's one but when we add another related for the other table this should be updated too.
```

Comment:
I don't think this should be updated, I'll look at the tests more closely when I review this later and understand why this might have changed.